packages/react/components/bookmark/src/components/Bookmark.tsx

```diff
@@ -45,38 +45,24 @@ const Styled = {
 export const Bookmark = () => {
     const { provider } = useBookmarkComponentContext();
 
-    const [loading, setLoading] = useState(true);
-    const [error, setError] = useState<Error | null>(null);
-
     const { value: bookmarks } = useObservableState(
         useMemo(() => provider?.bookmarks$ || EMPTY, [provider]),
-        { initial: provider?.bookmarks },
     );
 
-    useEffect(() => {
-        const sub = provider?.getAllBookmarks().subscribe({
-            complete: () => {
-                setLoading(false);
-            },
-            error: (err) => {
-                setError(err);
-            },
-        });
-        return () => sub?.unsubscribe();
-    }, [provider]);
+    const { value: isLoading } = useObservableState(
```

Comment:
Can we have a discussion about this?  I agree that this will work fro now but i would  like something in the line of this: 

Note: this is dummy code :P
```TS
const chechBookmarkLaodingStatus = (status) => !!status.has('fetch_bookmarks');


const {value: bookmarks, isLoading } = useObservableQueryState({
    queryState: useMemo(() => provider?.bookmarks$ || EMPTY, [provider]),
    isLaoding: useMemo((provider?.status$ || EMPTY).pipe(map((chechBookmarkLaodingStatus)))),
});
```
I feel its more clean. Byt requires a new useObservableQueryState hook.